addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/handler/PartitionThingHandler.java

```diff
@@ -155,22 +155,22 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     int partitionNumber = getPartitionNumber();
                     if (command.toString().equals("0")) {
                         dscAlarmBridgeHandler.sendCommand(DSCAlarmCode.PartitionDisarmControl, String.valueOf(partitionNumber));
-                        updateProperties(channelUID, 0, "Partition Disarmed");
+                        // updateProperties(channelUID, 0, "Partition Disarmed");
```

Comment:
They are part of one of the bug fixes.  I will remove those lines.  Actually, properties in this case is referring to a class called 'DSCAlarmProperties' which is basically a holdover from OH1 and was used to cross reference DSC Alarm properties to items.  I was using it in similar manner but with channels in OH2, but after closer examination I believe it can be removed.